server/passport/passportConfig.js

```diff
@@ -1,4 +1,4 @@
-const { CLIENT_SLACK_ID, CLIENT_SLACK_SECRET } = process.env;
+const { CLIENT_SLACK_ID, CLIENT_SLACK_SECRET,CLIENT_SLACK_CALLBACK } = process.env;
```

Comment:
This is to support multiple callbacks from SSO through Slack.
If not provided in the environment : The default callback would be to localhost